application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2159,6 +2228,16 @@ type Stop implements Node & PlaceInterface {
   zoneId: String
 }
 
+"Stop call represents the time when a specific trip on a specific date arrives to and/or departs from a specific stop location."
+type StopCall {
+  "Real-time estimates for arrival and departure times for this stop location."
+  realTime: CallRealTime
+  "Scheduled arrival and departure times for this stop location."
+  schedule: CallSchedule
+  "The stop where this arrival/departure happens."
+  stopLocation: CallStopLocation!
```

Comment:
What is the naming convention you decided on, naming the union by the "aggregate" `Call` or the wrapping type `StopCall`.  In this case `CallStopLocation` or `StopCallStopLocation`.

Reply:
I'm not sure if we have made a clear decision on this. It depends a bit on if we think there will be some other need for stop locations within the call scope or not. I think mostly we would have the same set of different types of stop locations for different use cases.